### PR TITLE
Added module 2 assignments of javascript

### PR DESCRIPTION
**Pull request by JuleeGunjal**

### CODE REVIEW

File: module_2/assignment1_1.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const PI = 3.147;
```
Comment (tejashreeSurveJosh) [inline module_2/assignment1_1.js:2]: Add comment what happen if we access const  and let before assignment

File: module_2/assignment1_2.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+let strZero = '0';
+let arr = [];
+let user = {
+  name: 'john',
+  age: 29,
+}
+let nan = NaN;
+let nullValue = null;
+
+console.log("float constant: ", typeof PI);
```
Comment (tejashreeSurveJosh) [inline module_2/assignment1_2.js]: Can you add comment as answer in front of all the console.log ?

File: module_2/assignment1_3.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+let arr = ['John', 'Ram', 'ari', 'nal'];
+const ARR = [3.147, 'Kiya', 23];
```
Comment (tejashreeSurveJosh) [inline module_2/assignment1_3.js]: Camel casing is required for defining an array, please change it.
All capital letter is use when some constant value is defined like PI / any constant message.

File: module_2/assignment1_3_obj.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+console.log(user); // { name: 'Julee', email: 'john1@gmail.com' }
+
+user = {
+  password: '12356',
+  birthDate: '25-06-1994',
+}  // Should replace the previous user by new one
+
+console.log(user); // { password: '12356', birthDate: '25-06-1994' }
+
+const USER = {
```
Comment (tejashreeSurveJosh) [inline module_2/assignment1_3_obj.js]: Please follow camel casing.

File: module_2/assignment1_1.js (hunk 1 above)
Comment (JuleeGunjal) [reply module_2/assignment1_1.js:2]: Done

File: module_2/assignment1_2.js (hunk 2 above)
Comment (JuleeGunjal) [reply module_2/assignment1_2.js]: done

File: module_2/assignment1_3.js (hunk 3 above)
Comment (JuleeGunjal) [reply module_2/assignment1_3.js]: done

File: module_2/assignment1_3_obj.js (hunk 4 above)
Comment (JuleeGunjal) [reply module_2/assignment1_3_obj.js]: done

**Pull request merged**